Code/Editor/EditorPreferencesDialog.h

```diff
@@ -18,6 +18,26 @@ namespace Ui
 }
 
 class EditorPreferencesTreeWidgetItem;
+void WidgetHandleKeyPressEvent(QWidget* widget, QKeyEvent* event)
```

Comment:
update: We should leave the definition in the .cpp file but keep the declaration in the .h - either as part of this PR or a follow-up we should then create a dummy widget to invoke this call in its own `keyPressEvent` and verify the behavior matches our expectations

Reply:
Changed.